tests/testthat/test-developer-to_integer.R

```diff
@@ -51,3 +51,14 @@ test_that("The statistcial information under null equals to event/4 udner equal
 
   expect_true(all(x$analysis$info0 - x$analysis$event / 4 == 0))
 })
+
+test_that("Validate the sample size rounding.", {
+
+  x <- gs_design_ahr(analysis_time = c(24, 36))
```

Comment:
Love to see these tests! Could we also add a test where ratio is not a positive integer? This particular example has `ratio=1`

```R
library("gsDesign2")
x <- gs_design_ahr(analysis_time = c(24, 36))
x$input$ratio
## [1] 1
```